app/boards/shields/tg4x/tg4x.keymap

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+#include <dt-bindings/zmk/bt.h>
+
+#define DEFAULT 0
+#define LOWER 1
+#define RAISE 2
+
+/ {
+    behaviors {
+        hm: homerow_mods {
+            compatible = "zmk,behavior-hold-tap";
+            label = "homerow mods";
+            #binding-cells = <2>;
+            tapping_term_ms = <225>;
+            flavor = "tap-preferred";
+            bindings = <&kp>, <&kp>;
+        };
+    };
+};
+
+/ {
+    keymap {
+        compatible = "zmk,keymap";
+
+        default_layer {
+            bindings = <
+    &kp GRAV   &kp Q       &kp W      &kp F         &kp P           &kp B    &kp J    &kp L      &kp U      &kp Y      &kp SCLN    &kp BKSP   
```

Comment:
Interesting keymap!

We try to have ZMK come with "stock" keymaps that are more basic/standardized, and then let user's override them with user configs to tweak to their liking.

Can we get a stock QWERTY version for here, and then I'm happy on Discord to help walk you through building locally with an "out of tree" keymap in a user config, so you can have the keymap you want.

Reply:
Sorry I was AWOL due to work. I will get this put together and submit this week.

Reply:
Revised keymap